docs/reference/commandline/network_create.md

```diff
@@ -22,6 +22,7 @@ Create a network
 
 Options:
       --attachable           Enable manual container attachment
+      --ingress              Specify the network provides the routing-mesh
```

Comment:
Can you also add this to the man-page? https://github.com/docker/docker/blob/master/man/src/network/create.md